tools/driver-cxx/driver-cxx.cpp

```diff
@@ -100,6 +102,18 @@ llvm::cl::opt<std::string> CacheDir(
     llvm::cl::desc("Where to store cache (defaults to /tmp/mull-cache)"),
     llvm::cl::cat(MullCXXCategory), llvm::cl::init("/tmp/mull-cache"));
 
+llvm::cl::opt<bool> ElementsReporterEnabled(
+    "reporter-elements", llvm::cl::Optional,
```

Comment:
Ideally, we should provide an option to enable several reporters, the same way as we do with mutators.

But we can do it separately.

Reply:
let's do it separately